source/_posts/2020-06-03-release-111.markdown

```diff
@@ -83,13 +83,16 @@ continue, resulting in a faster startup.
 
 - The OpenZwave beta integration is moving forward! Support for climate, fans
   and locks is added this release!
-- [@gadgetmobile] when all out on the Blebox integration, adding support for
+- [@gadgetmobile] went all out on the Blebox integration, adding support for
   a lot of platforms!
 - Google Assistant now supports using a select helper (aka `input_select`),
   amazing work [@ZephireNZ]!
 - [@frenck] added two new built-in Home Assistant events, helpful for
   automations: `automation_reloaded` and `scene_reloaded`. Using this as a
   trigger can be used for, e.g., re-applying a scene when it was changed.
+- The Plugwise integration has been improved by [@bouwew] and [@CoMPaTech],
```

Comment:
The user bouwew isn't listed in any contribution, hence it cannot be linked.

Reply:
It isn't allowed to attribute credit where due manually? (From the above comment fully understood - and maybe I'd need to revert the part mentioning him and summarizing the PRs in breaking as well then?)

Reply:
Sure, but it isn't linking now 😉 

Add `[@bouwew]: https://github.com/bouwew` to that same paragraph (at the end)